### PR TITLE
chore: disable `no-continue` eslint rule

### PR DESCRIPTION
**Pull request by nedsalk**
# Summary

Disabled `no-continue` rule and removed its usages after majority agreed it's okay to use it.

# Checklist

- [x] All **changes** are **covered** by **tests** (or not applicable)
- [x] All **changes** are **documented** (or not applicable)
- [x] I **reviewed** the **entire PR** myself (preferably, on GH UI)
- [x] I **described** all **Breaking Changes** (or there's none)

### CODE REVIEW

File: packages/utils/src/utils/toUtf8String.ts [hunk 1]
```diff
@@ -47,13 +46,13 @@ function onError(
   reason: Utf8ErrorReason,
   offset: number,
   bytes: Uint8Array,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (maschad) [inline packages/utils/src/utils/toUtf8String.ts:50]: Why are you removing these _eslint disable_ modifiers? They don't seem related to the PR

Comment (nedsalk) [reply packages/utils/src/utils/toUtf8String.ts:50]: No idea how this got removed :man_shrugging:
Fixed in https://github.com/FuelLabs/fuels-ts/pull/3225/commits/57781b52bfac3dd8a8af9354f0daa18d11536ad1

**Review by maschad — DISMISSED** [review]

File: .changeset/thin-waves-care.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+---
+
+chore: disable `no-continue` eslint rule
```
Comment (danielbate) [inline .changeset/thin-waves-care.md:4]: Need to change the target branch.

**Review by danielbate — APPROVED** [review]

**Review by maschad — APPROVED** [review]

**Review by petertonysmith94 — APPROVED** [review]

**Review by danielbate — APPROVED** [review]

**Review by arboleya — APPROVED** [review]
